maintenance/client_dependencies.py

```diff
@@ -0,0 +1,61 @@
+import re
+import subprocess
+from typing import List
+
+from shared import fetcher_internal
+
+
+PATH = 'shared_web/templates/jsdependencies.mustache'
+
+def run() -> None:
+    tags = [fetch_script_tag(library) + '\n' for library in get_dependencies()]
+    output = ''.join(tags)
+    write_dependencies(output)
+    subprocess.call(['git', 'add', PATH])
+    if subprocess.call(['git', 'commit', '-m', '"Update client dependencies."']) == 0:
+        subprocess.call(['git', 'push'])
+        subprocess.call(['hub', 'pull-request', '-b', 'master', '-m', 'Update client dependencies.' '-f'])
```

Comment:
This is the part I'm really unsure of.

Reply:
This should be an ad-hoc, and should never run on prod.
I can wrap it into the jenkins stuff

Reply:
Ah, yeah, that makes so much more sense.

Reply:
0292c632 does this.